nixos/tests/web-apps/movim/ejabberd-h2o.nix

```diff
@@ -0,0 +1,318 @@
+import ../../make-test-python.nix (
```

Comment:
Please use runTest

Reply:
The relevant issue is currently pinned https://github.com/NixOS/nixpkgs/issues/386873